### PR TITLE
log warning when error connecting to placement service

### PR DESCRIPTION
**Pull request by patrickxiong**
# Description

Change the log level from debug to warn when error connecting to placement service. 
## Issue reference

Please reference the issue this PR will close: https://github.com/dapr/dapr/issues/2896

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: pkg/actors/internal/placement.go [hunk 1]
```diff
@@ -355,7 +355,7 @@ func (p *ActorPlacement) establishStreamConn() (v1pb.Placement_ReportDaprStatusC
 		conn, err := grpc.Dial(serverAddr, opts...)
 	NEXT_SERVER:
 		if err != nil {
-			log.Debugf("error connecting to placement service: %v", err)
+			log.Warnf("error connecting to placement service: %v", err)
```
Comment (daixiang0) [inline pkg/actors/internal/placement.go:358]: Could you tell me the reason of this change?

Comment (patrickxiong) [reply pkg/actors/internal/placement.go:358]: It's for the issue
https://github.com/dapr/dapr/issues/2896

When the placement service cannot be connected, maybe we should log a warning.

**Review by artursouza — DISMISSED** [review]

**Comment by patrickxiong** [comment]
@artursouza Please approve the pull request again. Sorry I checked in an other change before the pull request finished.
I will open a new issue for the other change.

**Comment by yaron2** [comment]
@patrickxiong thanks for the PR. Before merging, I want to test that it doesn't spam the log output in case the connection breaks.

File: pkg/actors/internal/placement.go (hunk 1 above)
Comment (artursouza) [reply pkg/actors/internal/placement.go:358]: I agree. There will be a new change to ignore placement if not endpoint is set for placement service.

**Review by artursouza — APPROVED** [review]

**Pull request closed without merging**